### PR TITLE
chore: release 0.11.7

### PR DESCRIPTION
**Pull request by digikata**
Release fluvio 0.11.7

### CODE REVIEW

**Review by EstebanBorai — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,34 @@
...
+* Rename mirroring cmds to home and remote ([#3959](https://github.com/infinyon/fluvio/issues/3959))
+* Rename SpuSocket to StreamSocket ([#3958](https://github.com/infinyon/fluvio/issues/3958))
+* Rename remote, edge and core to mirror, remote and home ([#3966](https://github.com/infinyon/fluvio/issues/3966))
+* Graceful period for stream interruption ([#3969](https://github.com/infinyon/fluvio/issues/3969))
+* Sc-to-sc mirroring connection ([#3946](https://github.com/infinyon/fluvio/issues/3946))
+* Allow enabling wasi for connectors ([#3977](https://github.com/infinyon/fluvio/issues/3977))
+* Set default offset flush interval to 2s ([#3976](https://github.com/infinyon/fluvio/issues/3976))
+* `v0.11.6` post-release ([#3955](https://github.com/infinyon/fluvio/issues/3955))
+
+### Removed
```
Comment (ajhunyady) [inline CHANGELOG.md:30]: why is this labeled as removed?

Comment (digikata) [reply CHANGELOG.md:30]: Not sure, but it's fixed now

**Review by fraidev — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,30 @@
...
+* Upadate http client, better client isolation ([#3980](https://github.com/infinyon/fluvio/issues/3980))
+
+### Other
+
+* Enable `repository_url` ([#3961](https://github.com/infinyon/fluvio/issues/3961))
+* Rename mirroring cmds to home and remote ([#3959](https://github.com/infinyon/fluvio/issues/3959))
+* Rename SpuSocket to StreamSocket ([#3958](https://github.com/infinyon/fluvio/issues/3958))
+* Rename remote, edge and core to mirror, remote and home ([#3966](https://github.com/infinyon/fluvio/issues/3966))
+* Graceful period for stream interruption ([#3969](https://github.com/infinyon/fluvio/issues/3969))
+* Sc-to-sc mirroring connection ([#3946](https://github.com/infinyon/fluvio/issues/3946))
```
Comment (fraidev) [inline CHANGELOG.md:25]: Maybe this should be in "Added"?

Comment (fraidev) [reply CHANGELOG.md:25]: Or maybe not, because it's not completed

**Review by nacardin — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (digikata) [reply CHANGELOG.md:25]: I think it's ok here since the final feature is not completed.

**Pull request merged**